### DISCUSSION TITLE
Calculating derivatives within PySR

### DISCUSSION THREAD

**Discussion by TadeuNP** [MilesCranmer/PySR#314] [Q&A] [answered]
Hi, 

When running SymbolicRegression.jl, I can set enable_autodiff=True and the computation of derivatives will work. However, when using eval_diff_tree_array from PySR, I get the following error: 

RuntimeError: <PyCall.jlwrap (in a Julia function called from Python)
JULIA: Found no differential operators. Did you forget to set `enable_autodiff=true` when creating the `OperatorEnum`?

Is there any of doing it from the PySR?  I can't find any way of introducing this option.

Thanks!

**Comment by MilesCranmer** [comment, marked as answer]
Hm, I think this is a bug. Let me move this to the issues.

Moved to #315 

---

Edit: fixed on PySR v0.12.2.
